ntbs-integration-tests/TransferPages/ActionTransferPageTests.cs

```diff
@@ -110,9 +113,10 @@ public async Task AcceptTransferAlert_SuccessfullyChangesTbServiceOfNotification
             // Assert
             Assert.NotNull(resultDocument.QuerySelector("#return-to-notification"));
             var overviewUrl = RouteHelper.GetNotificationPath(id, NotificationSubPaths.Overview);
-            var overviewPage = await GetDocumentForUrl(overviewUrl);
+            var overviewPage = await GetDocumentForUrlAsync(overviewUrl);
             Assert.Contains("Abingdon Community Hospital", overviewPage.QuerySelector("#banner-tb-service").TextContent);
             Assert.Contains("TestCase TestManager", overviewPage.QuerySelector("#banner-case-manager").TextContent);
+            Assert.Contains("ABINGDON COMMUNITY HOSPITAL", overviewPage.QuerySelector("#overview-hospital-name").TextContent);

```

Comment:
if this and "banner-tb-service" are related can we pull this value into const and compare with toUpper if casing is important?

Reply:
They happen to be related in this case where the tb service and hospital is the same, I think we we're hard coding a lot of the test cases I'd prefer to keep them separate here